### PR TITLE
Objects And References - Answers and Tests Files

### PR DESCRIPTION
**Pull request by cerilacanda**

### CODE REVIEW

File: answers/ObjectsAndReferences/28dc6250-a538-4656-9809-c119767c7d0c/MissingNewKeywordTest.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        echo "This is a student class.\n";
+    }
+}
+
+$objectVarA = new Person();
+$objectVarB = $objectVarA;
+$objectVarC = &$objectVarA;
+$objectVarC->display();
+$objectVarA = new Student();
+$objectVarC->display();
```
Comment (TenshiLyle) [inline answers/ObjectsAndReferences/28dc6250-a538-4656-9809-c119767c7d0c/MissingNewKeywordTest.php]: Let us change the variable names.
```
$person = new Person();
$personRef = $person;
$anotherPersonRef = &$person;
$anotherPersonRef->display();
$person = new Student();
$anotherPersonA->display();
```

Comment (TenshiLyle) [reply answers/ObjectsAndReferences/28dc6250-a538-4656-9809-c119767c7d0c/MissingNewKeywordTest.php]: What is the purpose of `$objectVarB`?

File: answers/ObjectsAndReferences/3e23cbe1-3872-4c7c-abf4-67cf15e395c2/IncorrectAmperandPositionTest.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    public function display()
+    {
+        echo "This is a student class.\n";
+    }
+}
+
+$objectVarA = new Person();
+$objectVarB = $objectVarA;
+$objectVarC = &$objectVarA;
+$objectVarA->display();
```
Comment (TenshiLyle) [inline answers/ObjectsAndReferences/3e23cbe1-3872-4c7c-abf4-67cf15e395c2/IncorrectAmperandPositionTest.php]: Same as above.

File: answers/ObjectsAndReferences/6a7d9398-bb5b-45f4-a675-534204be8645/IncorrectObjectOperatorTest.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    public function display()
+    {
+        echo "This is a student class.\n";
+    }
+}
+
+$objectVarA = new Person();
+$objectVarB = $objectVarA;
+$objectVarC = &$objectVarA;
+$objectVarA->display();
```
Comment (TenshiLyle) [inline answers/ObjectsAndReferences/6a7d9398-bb5b-45f4-a675-534204be8645/IncorrectObjectOperatorTest.php]: And this

File: answers/ObjectsAndReferences/6d2e5aa8-9b8a-4a37-ad2b-e4229e44579f/MissingDollarSignTest.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        echo "This is a student class.\n";
+    }
+}
+
+$objectVarA = new Person();
+$objectVarB = $objectVarA;
+$objectVarC = &$objectVarA;
+$objectVarC->display();
+$objectVarA = new Student();
+$objectVarC->display();
```
Comment (TenshiLyle) [inline answers/ObjectsAndReferences/6d2e5aa8-9b8a-4a37-ad2b-e4229e44579f/MissingDollarSignTest.php]: Also this one.

File: answers/ObjectsAndReferences/8966cdec-0047-4deb-87a9-ce6b119402c4/MissingSemicolonTest.php [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    public function display()
+    {
+        echo "This is a student class.\n";
+    }
+}
+
+$objectVarA = new Person();
+$objectVarB = $objectVarA;
+$objectVarC = &$objectVarA;
+$objectVarA->display();
```
Comment (TenshiLyle) [inline answers/ObjectsAndReferences/8966cdec-0047-4deb-87a9-ce6b119402c4/MissingSemicolonTest.php]: And this one. Please check the rest of the content.

**Review by TenshiLyle — CHANGES_REQUESTED** [review]

File: answers/ObjectsAndReferences/28dc6250-a538-4656-9809-c119767c7d0c/MissingNewKeywordTest.php (hunk 1 above)
Comment (cerilacanda) [reply answers/ObjectsAndReferences/28dc6250-a538-4656-9809-c119767c7d0c/MissingNewKeywordTest.php]: @TenshiLyle `$objectVarB` in this example shows that it references or points to the same `Person` object that `$objectVarA` does.

> Let us change the variable names.
> 
> ```
> $person = new Person();
> $personRef = $person;
> $anotherPersonRef = &$person;
> $anotherPersonRef->display();
> $person = new Student();
> $anotherPersonA->display();
> ```

This is noted. I will the content related to this one. Thank you.

**Review by TenshiLyle — APPROVED** [review]

**Pull request merged**